lametro/templates/event/_event_header_live_public_comment.html

```diff
@@ -53,78 +53,41 @@
         {% endif %}
     </section>
 
-    {% if has_agenda and not event.has_passed %}
+    {% if has_agenda and event_notices %}
         <article class="col-md-5">
-            <h2 class="h4">Submit public comment remotely</h2>
+            {% if not event.has_passed %}
+                {% for notice in event_notices %}
+                    {% if "upcoming" in notice.broadcast_conditions and event.is_upcoming %}
+                        {{ notice.message|safe }}
+                    {% elif "ongoing" in notice.broadcast_conditions and event.is_ongoing %}
+                        {{ notice.message|safe }}
```

Comment:
This pattern makes it so that notices only ever show if the event's broadcast status matches the notifications' conditions. This is repeated across the `public_comment` partials